keras_cv/visualization/utils.py

```diff
@@ -0,0 +1,56 @@
+from typing import Dict
+from typing import List
+from typing import Tuple
+from typing import Union
+
+import tensorflow as tf
+
+E = Union[str, Dict[str, Union[str, int]]]
```

Comment:
Let's just drop the type annotations.

Reply:
ok :(

Reply:
Haha - sorry just trying to keep it consistent across the codebase for now!

Reply:
Maybe we will be using them in the future, but I'd like to keep it consistent.